esp/templates/program/modules/studentclassregmodule/catalog.html

```diff
@@ -153,6 +167,19 @@ <h2 style="text-align: center;">
 hideClassesOutOfGradeRange(student_grade);
 </script>
 
+<script type="text/javascript">
+$j("#grade_filter").change(function(){
```

Comment:
So, I'm having a hard time figuring out where `student_grade` in the above script actually comes from?